modules/session-storage/pages/udf.adoc

```diff
@@ -0,0 +1,144 @@
+= Using User-Defined Functions with a Session Store
+:page-aliases: tutorials:session-storage-tutorial:udf
```

Comment:
```suggestion
```

Is that `:page-aliases:` needed? It's normally used to have a redirect, this one would suggest to redirect from this page https://docs.couchbase.com/tutorials/session-storage-tutorial/udf.html which doesn't exist.

Reply:
I have no idea why I put a page-alias in there. Cargo culting I guess? If it's safe to remove, I'll take it out in #9